src/index.ts

```diff
@@ -1,5 +1,9 @@
 declare var HIFI_API_VERSION: string;
 
+// Check for browser compatibility
+import { checkBrowserCompatibility } from "./utilities/HiFiUtilities";
+checkBrowserCompatibility();
```

Comment:
please make sure this only runs in the browser context, not in NodeJS!

Reply:
Also, please add an `exports.checkBrowserCompatibility = checkBrowserCompatibility;` for external users :)

Reply:
Added